src/test/cypress/support/pageobjects/assessment/AbstractExerciseAssessmentPage.ts

```diff
@@ -0,0 +1,42 @@
+import { PUT, BASE_API } from './../../constants';
+/**
+ * Parent class for all exercise assessment pages.
+ */
+export abstract class AbstractExerciseAssessmentPage {
+    getInstructionsRootElement() {
+        return cy.get('[jhitranslate="artemisApp.textAssessment.instructions"]').parents('.card');
+    }
+
+    addNewFeedback(points: number, feedback?: string) {
+        cy.get('.btn-success').contains('Add new Feedback').click();
+        cy.get('.col-lg-6 >>>> :nth-child(1) > :nth-child(2)').clear().type(points.toString());
+        if (feedback) {
+            cy.get('.col-lg-6 >>>> :nth-child(2) > :nth-child(2)').type(feedback);
```

Comment:
I know these werent added in this PR but have you checked if there might be some better selectors?

Reply:
I think I had a look and couldn't directly find a better selector, but I will look again

Reply:
I updated the selectors